recipe_scrapers/_schemaorg.py

```diff
@@ -112,11 +112,11 @@ def ingredients(self):
 
     def nutrients(self):
         nutrients = self.data.get("nutrition")
```

Comment:
So if `self.data` doesn't have the key `"nutrition"` then `nutrients` will be `None`. Trying to iterate over `None`, I believe even calling `items()` on it, will cause an error.

If we can be assured that the key will always exist then I don't have a problem with this necessarily. 

I'll approve this and leave it to your judgment.

Reply:
I changed the default value to be an empty dict instead in case that does come up.  It is possible that that case would occur.